### PR TITLE
Implement velocity 

### PR DESCRIPTION
**Pull request by marcasmar1**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -89,6 +89,8 @@ pub struct Record {
     pub alarm: Option<Alarm>,
     /// The UAS location, which may be given in one of several declaration types.
     pub location: Location,
+    /// The UAS velocity given in ENU coordinates (relative to the UAS and given in meters per second).
```
Comment (aleokdev) [inline src/lib.rs]: What does "relative to the UAS" mean in this context? It's a velocity so it can't be relative to itself.

File: src/lib.rs [hunk 2]
```diff
@@ -120,6 +122,18 @@ pub enum Classification {
     Other,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Copy)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+/// Local cartesian coordinates given in East, North and Up using WGS84.
+pub struct CoordENU {
+    /// Component tangent to the parallels.
+    pub east: f64,
+    /// Component to meridians.
```
Comment (aleokdev) [inline src/lib.rs]:
```suggestion
    /// Component tangent to meridians. (Positive = north, negative = south)
```

File: src/lib.rs [hunk 3]
```diff
@@ -120,6 +122,18 @@ pub enum Classification {
     Other,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Copy)]
+#[cfg_attr(feature = "schemars", derive(JsonSchema))]
+/// Local cartesian coordinates given in East, North and Up using WGS84.
+pub struct CoordENU {
+    /// Component tangent to the parallels.
```
Comment (aleokdev) [inline src/lib.rs]:
```suggestion
    /// Component tangent to the parallels. (Positive = east, negative = west)
```

File: src/lib.rs [hunk 4]
```diff
@@ -120,6 +122,18 @@ pub enum Classification {
...
+    pub east: f64,
+    /// Component to meridians.
+    pub north: f64,
+    /// Component perpendicular to the WGS84 ellipsoid.
```
Comment (aleokdev) [inline src/lib.rs]:
```suggestion
    /// Component perpendicular to the WGS84 ellipsoid. (Positive = up, negative = down)
```

File: src/lib.rs (hunk 1 above)
Comment (marcasmar1) [reply src/lib.rs]: Relative to the UAS means that the ENU coordinates are centered on the UASs position, if that makes sense. Let me know if this shouldn't be this way.

Comment (aleokdev) [reply src/lib.rs]: That would make sense for a position, but I don't see how that would add any information to a velocity.

File: src/lib.rs [hunk 5]
```diff
@@ -89,6 +89,8 @@ pub struct Record {
     pub alarm: Option<Alarm>,
     /// The UAS location, which may be given in one of several declaration types.
     pub location: Location,
+    /// The UAS velocity given in ENU coordinates (relative to the UAS position and given in meters per second).
+    pub velocity: Option<CoordENU>,
```
Comment (aleokdev) [inline src/lib.rs:94]:
```suggestion
    #[serde(default)]
    pub velocity: Option<CoordENU>,
```
If velocity is not included in the input, ignore it. https://serde.rs/attr-default.html

**Pull request merged**